assets/docs/sayakpaul/collections/mlp-mixer/1.md

```diff
@@ -29,9 +29,20 @@ Several model variants are available:
 
 For more details on the training protocols, please follow [1, 3].
 
-The original model classes and weights [4] using the `jax2tf` tool [5]. For details on the conversion process,
+The original model classes and weights [4] were converted using the `jax2tf` tool [5]. For details on the conversion process,
 please refer to the [`conversion.ipynb`](https://colab.research.google.com/github/sayakpaul/MLPMixer-jax2tf/blob/main/conversion.ipynb) notebook.
 
+The author of this collection independently validated two models on the ImageNet-1k validation set. The table
+below reports the top-1 accuracies along with their respective logs from tensorboard.dev.
+
+| **Model** | **Top-1 Accuracy** | **tb.dev link** |
```

Comment:
Since I have explicitly mentioned where the models come from and how they were converted I think it would be easy for readers to navigate. Also, since the number of these models is pretty limited that might be helpful too. 

But I have mentioned the "pre-trained on I21k" whenever applicable in the individual model pages.